pkg/util/azureclient/apiversions.go

```diff
@@ -19,7 +19,7 @@ var apiVersions = map[string]string{
 	"microsoft.compute/snapshots":              "2020-05-01",
 	"microsoft.containerregistry":              "2020-11-01-preview",
 	"microsoft.resources/deployments":          "2021-04-01",
-	"microsoft.documentdb":                     "2021-01-15",
+	"microsoft.documentdb":                     "2023-04-15",
```

Comment:
How did you come up with this apiversion?
I just checked this [Documentation Page](https://learn.microsoft.com/en-us/azure/templates/microsoft.documentdb/databaseaccounts?pivots=deployment-language-arm-template) and didn't see this particular version there.

Reply:
There is a drop down menu for API Versions which has this version mentioned. 
[Here](https://learn.microsoft.com/en-us/azure/templates/microsoft.documentdb/2023-04-15/databaseaccounts?pivots=deployment-language-arm-template) you go.